hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/replicas/KeyVerificationResult.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.debug.replicas;
+
+import java.util.List;
+import org.apache.hadoop.hdds.protocol.DatanodeDetails;
+
+/**
+ * Class to hold the verification results for a key.
+ */
+public class KeyVerificationResult {
+  private final boolean keyPass;
```

Comment:
I don't think this is in the json output currently but we probably want a `keyCompleted` field as well which would map to the AND of all block verifier's `completed` statuses. This is used to indicate that a check failed without actually being able to run the check (like datanode not reachable). We would also set this to false if the `getKeyInfo` call fails.